Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
For sharing 1 copy, if there is no possible way, I think the simple way is to export source to 2 folder.
For 2 copies, that means we don't need to copy rule for "fennec" to desktop. We can use the application property in ua_overrides.jsm. May we can have a good way, like use macro/define to preprocess, then ua_overrider.jsm don't need to filter in initialization stage.